setup.py

```diff
@@ -29,19 +29,24 @@ def run(self):
         raise SystemExit(errno)
 
 
-setup(name='sklearn-pandas',
+setup(name='sklearn-pandas2',
```

Comment:
If we change the version number probably we don't need to change the name. Otherwise people will have to move from `sklearn-pandas==1.8.0` to `sklearn-pandas2==2.0.0`.